x-pack/plugins/security_solution/common/api/endpoint/workflow_insights/workflow_insights.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema, type TypeOf } from '@kbn/config-schema';
+
+const arrayWithNonEmptyString = (field: string) =>
+  schema.arrayOf(
+    schema.string({
+      minLength: 1,
+      validate: (id) => {
+        if (id.trim() === '') {
+          return `${field} can not be an empty string`;
+        }
+      },
+    })
+  );
+
+export const UpdateWorkflowInsightRequestSchema = {
+  params: schema.object({
+    insightId: schema.string({
+      minLength: 1,
+      validate: (id) => {
+        if (id.trim() === '') {
+          return 'insightId can not be an empty string';
+        }
+      },
+    }),
+  }),
+  body: schema.object({
```

Comment:
is this hitting our internal API? Do you think it makes sense to enable accepting `unknown` values too?

Reply:
I don’t see the need for it at the moment